docs/self-managed/platform-deployment/helm-kubernetes/guides/air-gapped-installation.md

```diff
@@ -0,0 +1,147 @@
+- [camunda/identity](https://hub.docker.com/r/camunda/identity)
+- [postgres](https://hub.docker.com/_/postgres)
+- [bitnami/keycloak](https://hub.docker.com/r/bitnami/keycloak)
+- [elasticsearch](https://hub.docker.com/_/elasticsearch)
+
+## Required Helm charts
+
+The following charts must be available in your air-gapped environment:
+
+- [Camunda Platform Helm chart](https://github.com/camunda/camunda-platform-helm)
+- [Elasticsearch Helm chart](https://github.com/elastic/helm-charts/tree/main/elasticsearch)
+- [Keycloak Helm chart](https://github.com/bitnami/charts/tree/master/bitnami/keycloak)
+- [Postgres Helm chart](https://github.com/bitnami/charts/tree/master/bitnami/postgresql/)
+
+## Dependencies explained
+
+Identity utilizes Keycloak and allows you to manage users, roles, and permissions for Camunda Platform 8 components. This third-party dependency is reflected in the Helm chart as follows:
+
+```
+camunda-platform
+    |_ elasticsearch
+    |_ identity
+        |_ keycloak
+            |_ postgresql
+    |_ zeebe
+    |_ optimize
+    |_ operate
+    |_ tasklist
+```
+
+- Keycloak is a dependency for Camunda Identity and PostgreSQL is a dependency for Keycloak.
+- Elasticsearch is a dependency for Zeebe, Operate, Tasklist, and Optimize.
+
+The values for the dependencies Keycloak and PostgreSQL can be set in the same hierarchy:
+
+```yaml
+identity:
+  [identity values]
+  keycloak:
+    [keycloak values]
+    postgresql:
+      [postgresql values]
+```
+
+## Push images to your repository
+
+All the [required images](#required-images) need to be pushed to your repository using the following steps:
+
+1. Tag your image using the following command (replace `<IMAGE ID>`, `<DOCKER REPOSITORY>`, and `<DOCKER TAG>` with the corresponding values.)
+
+```
+docker tag <IMAGE_ID> example.jfrog.io/camunda/<DOCKER_IMAGE>:<DOCKER_TAG>
+```
+
+2. Push your image using the following command:
+
+```
+docker push example.jfrog.io/camunda/<DOCKER_IMAGE>:<DOCKER_TAG>
+```
+
+## Deploy Helm charts to your repository
+
+You must deploy the [required Helm charts](#required-helm-charts) to your repository.
+For example, in JFrog you can do this via the Artifactory's REST API.
+
+```
+curl -u <USER>:<PASSWORD> -T <PATH_TO_FILE> "https://example.jfrog.io/artifactory/camunda-platform/<TARGET_FILE_PATH>"
+```
+
+Alternatively, you can set up remote repositories (for example, with URL [https://helm.camunda.io](https://helm.camunda.io) for Camunda charts, [https://helm.elastic.co](https://helm.elastic.co) for Elastic).
+
+### Add your Helm repositories
+
+You must add your Helm chart repositories to use the charts:
+
+```
+helm repo add camunda https://example.jfrog.io/artifactory/api/helm/camunda-platform
+helm repo add elastic https://example.jfrog.io/artifactory/api/helm/elastic
+helm repo add bitnami https://example.jfrog.io/artifactory/api/helm/bitnami
+helm repo update
+```
+
+### Helm Chart values
```

Comment:
Since we don't capitalize all words in the titles, then it should be `chart` (non-capitalized) according to [Helm's convention](https://helm.sh/docs/chart_best_practices/conventions/#usage-of-the-words-helm-and-chart).